src/client/visualizations/scatterplot/x-axis.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2017-2022 Allegro.pl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import * as React from "react";
+
+import "./scatterplot.scss";
+
+import { Stage } from "../../../common/models/stage/stage";
+import { Unary } from "../../../common/utils/functional/functional";
+import { roundToHalfPx } from "../../utils/dom/dom";
+import { LinearScale } from "../../utils/linear-scale/linear-scale";
+
+const TEXT_OFFSET_X = 16;
+
+interface XAxisProps {
+  stage: Stage;
+  ticks: number[];
+  tickSize: number;
+  scale: LinearScale;
+  formatter: Unary<number, string>;
+}
+
+export const XAxis: React.SFC<XAxisProps> = ({ stage, ticks, scale, formatter, tickSize }) => {
+  const labelY = tickSize + TEXT_OFFSET_X;
+  const linePositionY = roundToHalfPx(0);
```

Comment:
What does round(0) mean?

Reply:
It is round to half, not just round. Result will be always 0.5, but method call show intent.